tests/kivaloo_servers.sh

```diff
@@ -86,19 +86,10 @@ lbs_stop() {
 	rm -f "${_lbs_stop_pidfile}"
 
 	# Wait for lbs server to stop.
-	while has_pid "lbs/lbs" ; do
-		if [ "${VERBOSE}" -ne 0 ]; then
-                        printf "Waiting to stop: lbs/lbs\n" 1>&2
-                fi
-                "${msleep}" 100
-	done
+	wait_while has_pid "lbs/lbs"
```

Comment:
Using `wait_while` in `tests/kivaloo_servers.sh` is in addition to the libcperciva import.